packages/jss-observable/package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "jss-observable",
+  "description": "JSS plugin for TC39 Observables support",
+  "version": "0.1.0",
+  "author": {
+    "name": "Oleg Slobodskoi",
+    "email": "oleg008@gmail.com"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git@github.com:cssinjs/jss.git"
+  },
+  "keywords": [
+    "jss",
+    "style",
+    "sheet",
+    "stylesheet",
+    "css",
+    "components",
+    "composable",
+    "css in js",
+    "css-in-js",
+    "observable",
+    "reactive",
+    "rxjs"
+  ],
+  "license": "MIT",
+  "main": "./lib/index.js",
+  "engines": {
+    "node": ">=4"
+  },
+  "scripts": {
+    "prepare": "node ../../scripts/prepare.js"
+  },
+  "peerDependencies": {
+    "jss": "^9.7.0"
+  },
+  "devDependencies": {
+    "jss": "^9.7.0"
+  },
+  "files": [
+    "dist",
+    "lib",
+    "src",
+    "readme.md",
+    "changelog.md",
+    "LICENSE"
+  ],
+  "dependencies": {
+    "is-observable": "^1.1.0"
```

Comment:
there was a problem with is-observable, see in the issues, thats why we have own version of this function. I think it is small enough to just be copied

Reply:
Yeah, just saw this on travis. Where should we put this function, because jss uses that in the `cloneStyle` function.

Reply:
Yeah I wonder how to separate it properly. Ideally it should be entirely encapsulated in jss-observable

Reply:
Yeah, also had that utility function inside `jss-observable` but that created the dependency cycle again and having it inside jss isn't right either. Could we create a PR to not use const in the `is-observable` package?

Reply:
no, is-observable package is for node only, see closed issues there, we talked to the maintainer

Reply:
one way is to create utilities package and have it there, another way is to duplicate this function, its one line

Reply:
Yeah, let's go with duplicating the line for now until we find a better way.

Reply:
yep